src/BinaryParsers/PEBinary/ProgramDatabase/LinkerCommandLine.cs

```diff
@@ -48,22 +74,55 @@ public LinkerCommandLine(string commandLine)
                     continue;
                 }
 
-                // There are multiple /debug options so use StartsWith
-                if (ArgumentStartsWith(argument, "debug"))
+                // There are multiple /debug options so use StartsWith.  There is also /debugtype: so we must be careful not
+                // to over-match against that.
+                if (ArgumentStartsWith(argument, "debug:") || ArgumentEquals(argument, "debug"))
                 {
                     debugSet = true;
+
+                    // "/DEBUG changes the defaults for the /OPT option from REF to NOREF and from ICF to NOICF, so if you
+                    //  want the original defaults, you must explicitly specify /OPT:REF or /OPT:ICF."
+                    // https://docs.microsoft.com/cpp/build/reference/debug-generate-debug-info?view=msvc-170#remarks
+                    this.OptimizeReferencesEnabled = false;
+                    this.COMDATFoldingEnabled = false;
                 }
-                else if (ArgumentEquals(argument, "opt:ref"))
-                {
-                    optRef = true;
-                }
-                else if (ArgumentEquals(argument, "opt:icf"))
-                {
-                    optIcf = true;
-                }
-                else if (ArgumentEquals(argument, "opt:lbr"))
+                else if (ArgumentStartsWith(argument, "opt"))
                 {
-                    optLbr = true;
+                    // "The /OPT arguments may be specified together, separated by commas. For example, instead of
+                    //  /OPT:REF /OPT:NOICF, you can specify /OPT:REF,NOICF."
```

Comment:
It was a bug that this parsing did not already support specifying multiple /OPT options with commas.